actions/plan/artifact-repositories/download-one.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+check_slug() {
+	local slug="${1}"; shift
+
+	if [[ ! "${slug}" =~ ^[a-zA-Z0-9][a-zA-Z0-9-]+/[a-zA-Z0-9_.-]+$ ]]; then
```

Comment:
Does this regex match what github is using to validate repo names?

Reply:
org/user names, not precisely - it's more restrictive
repo names, should be about right, yeah

The ref names in check_ref, definitely not, but didn't want to think about it too hard, can be relaxed in the future if needed